new-standalone-service.sh

```diff
@@ -0,0 +1,108 @@
+
+cp -R frameworks/template $PROJECT_PATH/$PROJECT_NAME
+cp -R tools $PROJECT_PATH/$PROJECT_NAME/tools
+
+rm -rf $PROJECT_PATH/$PROJECT_NAME/build
+rm -rf $PROJECT_PATH/$PROJECT_NAME/cli/dcos-*/*.whl
+rm -rf $PROJECT_PATH/$PROJECT_NAME/cli/dcos-*/dcos-*
+rm -rf $PROJECT_PATH/$PROJECT_NAME/cli/python/{build,dist}
+rm -rf $PROJECT_PATH/$PROJECT_NAME/build.sh
+
+cat > $PROJECT_PATH/$PROJECT_NAME/build.sh <<'EOF'
+#!/bin/bash
+set -e
+
+# capture anonymous metrics for reporting
+curl https://mesosphere.com/wp-content/themes/mesosphere/library/images/assets/sdk/build-sh-start.png >/dev/null 2>&1
+
+FRAMEWORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+BUILD_DIR=$FRAMEWORK_DIR/build/distributions
+PUBLISH_STEP=${1-none}
+export REPO_NAME="$(basename $FRAMEWORK_DIR)"
+export BUILD_BOOTSTRAP=no
+export TOOLS_DIR=${FRAMEWORK_DIR}/tools
+export CLI_DIR=${FRAMEWORK_DIR}/cli
+export ORG_PATH=github.com/$REPO_NAME
+${FRAMEWORK_DIR}/tools/build_framework.sh $PUBLISH_STEP $REPO_NAME $FRAMEWORK_DIR $BUILD_DIR/$REPO_NAME-scheduler.zip
+
+# capture anonymous metrics for reporting
+curl https://mesosphere.com/wp-content/themes/mesosphere/library/images/assets/sdk/build-sh-finish.png >/dev/null 2>&1
+
+EOF
+
+chmod +x $PROJECT_PATH/$PROJECT_NAME/build.sh
+mv $PROJECT_PATH/$PROJECT_NAME/cli/dcos-template $PROJECT_PATH/$PROJECT_NAME/cli/dcos-$PROJECT_NAME
+mv $PROJECT_PATH/$PROJECT_NAME/src/main/java/com/mesosphere/sdk/template/ $PROJECT_PATH/$PROJECT_NAME/src/main/java/com/mesosphere/sdk/$PROJECT_NAME/
+mv $PROJECT_PATH/$PROJECT_NAME/src/test/java/com/mesosphere/sdk/template/ $PROJECT_PATH/$PROJECT_NAME/src/test/java/com/mesosphere/sdk/$PROJECT_NAME/
+
+if [ $? -ne 0 ]; then
+    echo "Failed to create new project"
+    cleanup
+    exit 1
+fi
+
+UPPER_CASE_PROJECT_NAME=$(echo $PROJECT_NAME | awk '{print toupper($0)}')
+
+find $PROJECT_PATH/$PROJECT_NAME -type f -exec sed -i.bak "s/template/$PROJECT_NAME/g; s/TEMPLATE/$UPPER_CASE_PROJECT_NAME/g; s/template/$PROJECT_NAME/g" {} \;
+sed -i.bak -e '21,$ d' $PROJECT_PATH/$PROJECT_NAME/src/main/dist/svc.yml
+find $PROJECT_PATH/$PROJECT_NAME -type f -name *.bak -exec rm {} \;
+
+sed -i.bak "s/compile project(\":scheduler\")/compile \"mesosphere:scheduler:0.12.0\"/g" $PROJECT_PATH/$PROJECT_NAME/build.gradle
+sed -i.bak "s/compile project(\":executor\")/compile \"mesosphere:executor:0.12.0\"/g" $PROJECT_PATH/$PROJECT_NAME/build.gradle
+sed -i.bak "s/testCompile project(\":testing\")/testCompile \"mesosphere:testing:0.12.0\"/g" $PROJECT_PATH/$PROJECT_NAME/build.gradle
+sed -i.bak '/distZip.dependsOn ":executor:distZip"/d' $PROJECT_PATH/$PROJECT_NAME/build.gradle
+sed -i.bak '/distZip.finalizedBy copyExecutor/d' $PROJECT_PATH/$PROJECT_NAME/build.gradle
+
+GRADLE_EXISTS="yes"
+command -v gradle >/dev/null 2>&1 || { echo >&2 "I require gradle but it's not installed. Please install gradle and then run 'gradle wrapper --gradle-version 3.4' inside $PROJECT_PATH/$PROJECT_NAME"; GRADLE_EXISTS="no"; }
```

Comment:
This could just use the `gradlew` command provided at the root of `dcos-commons`. I was able to build a wrapper like this just fine:

`myproject$ path/to/dcos-commons/gradlew wrapper --gradle-version 3.4`